Dockerfile.release

```diff
@@ -0,0 +1,14 @@
+# vim: syntax=dockerfile
+FROM alpine AS servok-grpc
+ARG TARGETARCH
+ARG GRPC_HEALTH_PROBE_VERSION=0.3.6
+RUN apk update && \
```

Comment:
grpc-health-probe has an official multiarch image we can use now, check out the spicedb release dockerfile: https://github.com/authzed/spicedb/blob/main/Dockerfile.release